agentops/client.py

```diff
@@ -288,7 +290,7 @@ def handle_exception(exc_type, exc_value, exc_traceback):
             # Then call the default excepthook to exit the program
             sys.__excepthook__(exc_type, exc_value, exc_traceback)
 
-        atexit.register(lambda: cleanup(end_state="Success", end_state_reason="Process exited normally"))
+        atexit.register(lambda: cleanup(end_state="Indeterminate", end_state_reason="Process exited normally"))
```

Comment:
Maybe change message to something else. "Normally" implies some intention, but the state is indeterminate